db/migrate/helpers/delete-content-item.rb

```diff
@@ -0,0 +1,21 @@
+module Helpers
+  module DeleteContentItem
+    def self.destroy_supporting_objects(content_items)
```

Comment:
Nice, can we update the readme to point to this? 
https://github.com/alphagov/publishing-api#deleting-content-items